includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -11878,6 +11902,7 @@ class AMP_Allowed_Tags_Generated {
 						'requires_usage' => 2,
 						'version' => array(
 							'0.1',
+							'1.0',
```

Comment:
This change broke a site. See https://wordpress.org/support/topic/google-optimize-integration-2/

The `1.0` version of `amp-experiment` is not stable yet (it's still itself experimental), necessitating sites to roll back to `0.1`.

Reply:
Opened issue to address: https://github.com/ampproject/amp-wp/issues/4628